tool/lib/commands/serve.dart

```diff
@@ -13,10 +13,14 @@ import 'package:path/path.dart' as path;
 import 'shared.dart';
 
 const _buildAppFlag = 'build-app';
+const _debugServerFlag = 'debug-server';
+
+// TODO: The following section of args are here just to pass the arg through to
```

Comment:
can you create an issue for the todo and link it here? `TODO(issue-link): consider using ...`